pkg/manifests/manifests.go

```diff
@@ -3720,3 +3724,16 @@ func hashSecretData(s *v1.Secret) string {
 
 	return strconv.FormatUint(h.Sum64(), 32)
 }
+
+// hashConfigMapData hashes the configmap's data and returns the hash value as a string.
```

Comment:
maybe make the above `hashSecretData` take a `map[string][]byte` (and change its name) to make it work for secrets and configmaps

Reply:
sounds good to me. Something like?

````
func hashByteMap(map[string][]byte) string {
// same as hashSecretData()
}

func hashStringMap(m map[string]string) string {
  byteMap := make(map[string][]byte, len(m)
  for k, v := range m {
    byteMap[k] = []byte(v)
  }
  return hashByteMap(byteMap)
}